lib/fastMerge.js

```diff
@@ -1,45 +1,69 @@
 // Mostly copied from https://medium.com/@lubaka.a/how-to-remove-lodash-performance-improvement-b306669ad0e1
 
+let mergeObject;
+
+/**
+* Is the object Mergeable
+*
+* @param {*} val
+* @returns {*|boolean}
+*/
```

Comment:
```suggestion
/**
* @param {mixed} val
* @returns {mixed}
*/
```
Don't include a method description unless it's valuable (and what you had there just repeats the method name). For the return value, maybe we should cast it to always return a boolean?